Newliquors

Ordonis opened the pull request:
:cl: Ordo
rscadd: Adds a few new liquors to the bar, and a few new cocktails to boot! 
/:cl:

Goddamn how did we not have Triple Sec as a beverage already the jokes are so obvious I'm disappointed in all of us. 

TODONE: give quintuple sec and quadruple sec benefits for security jobs a la nothing for the mime
TODO (noncritical): Hope desperately that I or someone else can make halfway decent sprites at somepoint
TODONE: add even more things to anger the clown

Drinks themselves are now ready for merging, unless people are adamant that they have custom sprites first.

Ordonis committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

Ordonis committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

Ordonis committed:
Merge remote-tracking branch 'refs/remotes/tgstation/master'

Ordonis committed:
Adds a few new liqueurs and cocktails

How the fuck in THIS game of all games do we not have Triple Sec.
SERIOUSLY.

Ordonis committed:
Well there goes a one commit clean PR rip me

reeeeee

Ordonis committed:
Adds sounds, fixes colors

Ordonis committed:
Fixes recipies giving eggnog

Don't ask

Anonmare commented on code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:
```diff
@@ -1298,3 +1298,80 @@ All effects don't start immediately, but rather get worse over time; the rate is
+/datum/reagent/consumable/ethanol/grasshopper
+	name = "Grasshopper"
+	id = "grasshopper"
+	description = "A fresh and sweet dessert shooter.Difficult to look manly while drinking."
```
Should be a space here,
"...shooter. Difficult to..."
instead of 
"...shooter.Difficult to..."

PKPenguin321 commented:
Malformed changelog, remove the :  behind the rscadd

Ordonis committed:
Adds space where space should be

Ordonis committed:
Travis stop stalling reeeeeee

PKPenguin321 commented:
You removed the wrong :

Ordonis committed:
Fixes spelling mistakes

Ordonis committed:
Adds some health benefits to sec drinks.

Quadruple sec is mild, but easy to make; quintuple sec is STRONG but
more difficult.

Ordonis committed:
Merge remote-tracking branch 'origin/Newliquors' into Newliquors

# Conflicts:
#	code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

Ordonis committed:
Fixes merge conflict annoyance

Ordonis committed:
FUCK

Tlaltecuhtli commented:
remove the 7 healing chem

coiax commented on code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:
```diff
@@ -1298,3 +1298,97 @@ All effects don't start immediately, but rather get worse over time; the rate is
+	description = "A minty liqueur excellent for refreshing, cool drinks."
+	color = "#00cc00"
+	boozepwr = 20
+	taste_description = "a minty, cool, and invigorating splash of cold streamwater"
```
Okay, this is a LITTLE bit wordy for a taste description.

praisenarsie replied:
its just fluff so it doesn't matter

coiax replied:
I mean, it does. Try to cut down the wordiness please.

ExcessiveUseOfCobblestone replied:
explain

coiax replied:
Most of the other alchohol tastes are pithier, with a few exceptions. These are all REALLY long.

Robustin replied:
Agreed, base reagents shouldn't given sentence-length flavors.

ShizCalev replied:
This isn't even the longest one lol

Personally I'm a fan of giving a bit more description to flavors beyond "tastes like egg and cream", or simply "tastes like cola"; This is an RPG after all.

Ordonis replied:
Honestly I was planning on running through all of the other beverages and foods at some point to add a little more fluff; service jobs tend to rely more heavily on roleplaying (perhaps botany excepted) than others, so by having people able to make Dwarf Fortress style absurd concoctions that truly express how mad the combination of flavors would be, I think there's at least a little more fun to be had. Similarly it adds in some new opportunities for masking poisons in taste descriptions; "you can taste unadulterated pain and fire with a hint of sweetness" could easily describe a chili... or cyanide, which, from what we know, has a strong burning sensation. I can see it's a contentious direction, though.

coiax replied:
I mean, there's a limited amount of "balance" in terms of flavours, because you don't want EVERY reagent to have a unique taste. Most medications and poisons just taste of bitterness, with some exceptions. All of the taste descriptions pretty much I ported from bay, with some modifications here and there. They're supposed to be used in compounds.

Right now reagents can't have multiple tastes (as in, hops = 3, bitterness = 1), they can only have a single one, which I think leads to more giant compound tastes.

Ordonis commented:
@Tlaltecuhtli Really? It's less powerful than Hearty Punch, and much harder to make (insofar as you need bananium for clown's tears), assuming you're referring to Quintuple Sec. If it's truly an issue, I can dial it back, but I think there should be some reward for bothering to make it, given how much cooperation it entails.

Robustin commented:
You don't need bananium for clowns tears but I think its fine. Maybe increase the booze power since its a concentrated drink and you can't "OD" on these chems - so if you've got an officer filled up with the stuff, make sure he's going to be absolutely trashed too.

Robustin commented on code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:
```diff
@@ -1298,3 +1298,97 @@ All effects don't start immediately, but rather get worse over time; the rate is
+	id = "quintuple_sec"
+	description = "Law, Order, and Police Brutality distilled into one single elixir of JUSTICE."
+	color = "#ff3300"
+	boozepwr = 40
```
Increase the boozepwr on this to maybe 80 or more. I like niche drinks but given the strength of this drink it needs some serious side-effects (plus you've got a seriously concentrated drink at this point). 

Plus there's something hilarious about a nigh-unkillable officer just stumbling around puking their guts out.

Ordonis replied:
I think that's an excellent balance direction. Quadruple sec is fairly mild, and is only two ~40abv drinks combined with a non-alcoholic cocktail, so I think keeping it at around 35ish boozepwr is fine (also it isn't going to make you Judge Dredd himself), but you're correct that Quintuple should make you Judge Dredd... at a price. Old Man Henderson Cop balance update incoming.

ShizCalev replied:
Nitpicking / applying real world logic to this, but this drink's ABV would be roughly around 31.75%, which is around the ballpark of an old-fashioned (which surprisingly we don't have.) 

The triple-sec would be the highest ABV ingredient at around 40% ABV, so this wouldn't make sense at even this strength (since you wouldn't be increase the amount of alcohol in something by combining everything, you'd be diluting it.)

Ordonis replied:
Honestly, that's one of the reasons I had it being lower to begin with; I'm not sure if boozepwr is supposed to directly correlate with abv values, but I was roughly going off of that originally. The current update is solely a balance thing; going through and fixing all of the different drinks to have boozepwr correlating to their real world abv would be something I'd be willing to do, but I suspect that boozepwr is largely a balance factor in most drinks.

Ordonis committed:
Updates boozepwr of Quadruple/Quintuple sec

If someone ever adds sex on the beach as a cocktail, I fully expect them
to add in Sex-tuple Sec as well.

Tlaltecuhtli commented:
is clowntears a reagent?

Ordonis committed:
Makes clowntears reagent explicit within food_reagents

Removed hacky solution, introduced better one

Ordonis commented:
It seemed to be in a weird pseudo-reagent state where if you got a container with it and the other ingredients it would work, but was inconsistent on grinding; before I had just been spawning the components inside a glass, this should make any potential problems with that moot.
